packages/ai-jsx/src/lib/util.ts

```diff
@@ -8,12 +8,22 @@ export function getEnvVar(name: string, shouldThrow: boolean = true) {
   // In some JS build environments, `process` is not defined. I'm not sure
   // how to update our types to reflect that.
   // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-  const process = globalThis.process || undefined;
+  let env = globalThis.process?.env ?? undefined;
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  if (env === undefined) {
+    // Webpack for Create React App requires `process.env`, but in other environments
+    // process might not be defined, so we need to try/catch it.
+    try {
+      env = process.env;
+    } catch {
+      // Do nothing, leave it undefined.
+    }
```

Comment:
Is there a reason we can't just do a `'env' in process` check? The `try/catch` feels a bit awkward.

Reply:
The tricky part is:
- webpack apparently needs to see the literal `process.env` to replace it with the build-time value
- some environments don't have `process` defined at all

...so this was the only way I could figure out how to do it.